client/app/bundles/course/assessment/submission/reducers/liveFeedback.js

```diff
@@ -1,125 +1,172 @@
 import { produce } from 'immer';
 
+import { suggestionsTranslations } from 'course/assessment/submission/translations';
+
 import actions from '../constants';
 
-const initialState = {
+export const LIVE_FEEDBACK_LOCAL_STORAGE_KEY = 'liveFeedbackStateLocalStorage';
+
+const loadStateFromLocalStorage = () => {
+  try {
+    const serializedState = localStorage.getItem(
+      LIVE_FEEDBACK_LOCAL_STORAGE_KEY,
+    );
+    return serializedState ? JSON.parse(serializedState) : undefined;
+  } catch (error) {
+    return undefined;
+  }
+};
+
+const saveStateToLocalStorage = (state) => {
+  const serializedState = JSON.stringify(state);
+  localStorage.setItem(LIVE_FEEDBACK_LOCAL_STORAGE_KEY, serializedState);
+};
+
+const initialState = loadStateFromLocalStorage() || {
   feedbackUrl: null,
   feedbackByQuestion: {},
 };
-export default function (state = initialState, action) {
+
+const getRandomSuggestions = () => {
+  const suggestionValues = Object.values(suggestionsTranslations);
+  const shuffled = suggestionValues.sort(() => 0.5 - Math.random());
+  // TODO: Temporarily change to show no suggestions
+  return shuffled.slice(0, 0);
+};
+
+const getOrDefault = (obj, paramKeys, defaultValue) => {
+  const keys = Array.isArray(paramKeys) ? paramKeys : [paramKeys];
+  return keys.reduce(
+    (acc, key) => (acc && acc[key] !== undefined ? acc[key] : defaultValue),
+    obj,
+  );
+};
+
+const getConversation = (state, questionId) =>
+  getOrDefault(state, ['feedbackByQuestion', questionId, 'conversation'], []);
+
+const updateFeedbackForQuestion = (draft, questionId, newFeedbacks) => {
+  draft.feedbackByQuestion[questionId] = {
+    ...getOrDefault(draft.feedbackByQuestion, questionId, {}),
+    ...newFeedbacks,
+  };
+  saveStateToLocalStorage(draft);
+};
```

Comment:
Your states are not being saved properly, please double-check.

e.g. leave the [Get Help] window in open state, navigate away, then navigate back, the window won't show up as open state.